chart.js.d.ts

```diff
@@ -0,0 +1,605 @@
+      pointBackgroundColor?: string | Array<string>
+      pointBorderWidth?: number | Array<number>
+      pointHoverRadius?: number | Array<number>
+      pointHoverBackgroundColor?: string | Array<string>
+      pointHoverBorderColor?: string | Array<string>
+      pointHoverBorderWidth?: number | Array<number>
+      tension?: number
+      yAxisID?: string
+    }>
+  }
+
+  interface LineChartOptions extends ChartOptions {
+    showLines?: boolean
+    stacked?: boolean
+    scales?: {
+      xAxes?: Array<ScaleOptions & {
+        id?: string
+      }>
+      yAxes?: Array<ScaleOptions & {
+        id?: string
+      }>
+    }
+  }
+
+  class LineChartInstance extends ChartInstance {
+    data: LineChartData;
+    options: LineChartOptions;
+  }
+
+  interface BarChartData extends ChartData {
+    datasets?: Array<ChartDataSet & {
+      label?: string
+      backgroundColor?: string | Array<string>
+      borderColor?: string | Array<string>
+      borderWidth?: number | Array<number>
+      hoverBackgroundColor?: string | Array<string>
+      hoverBorderColor?: string | Array<string>
+      yAxisID?: string
+    }>
+  }
+
+  interface BarChartOptions extends ChartOptions {
+    stacked?: boolean
+    scales?: Array<{
+      xAxes?: Array<ScaleOptions & {
+        position?: string
+        id?: string
+        categoryPercentage?: number
+        barPercentage?: number
+        gridLines?: Array<ScaleGridLinesOptions & {
+          offsetGridLines?: boolean
+        }>
+      }>
+      yAxes?: Array<ScaleOptions & {
+        position?: string
+        id?: string
+      }>
+    }>
+  }
+
+  class BarChartInstance extends ChartInstance {
+    data: BarChartInstance;
+    options: BarChartOptions;
+  }
+
+  interface RadarChartData extends ChartData {
+    datasets?: Array<ChartDataSet & {
+      label?: string
+      backgroundColor?: string
+      borderColor?: string
+      pointBackgroundColor?: string
+      pointBorderColor?: string
+      pointHoverBackgroundColor?: string
+      pointHoverBorderColor?: string
+    }>
+  }
+
+  interface RadarChartOptions extends ChartOptions {
+    scale?: Array<ScaleOptions>
+  }
+
+  class RadarChartInstance extends ChartInstance {
+    data: RadarChartData;
+    options: RadarChartOptions;
+  }
+
+  interface PolarAreaChartData extends ChartData {
+    datasets?: Array<ChartDataSet & {
+      label?: string
+      backgroundColor?: Array<string>
+    }>
+  }
+
+  interface PolarAreaChartOptions extends ChartOptions {
+    scale?: Array<ScaleOptions & {
+      lineArc?: boolean
+    }>
+    animation?: {
+      animateRotate?: boolean
+      animateScale?: boolean
+    }
+  }
+
+  class PolarAreaChartInstance extends ChartInstance {
+    data: PolarAreaChartData;
+    options: PolarAreaChartOptions;
+  }
+
+  interface CircularChartData extends ChartData {
+    datasets?: Array<ChartDataSet & {
+      backgroundColor?: Array<string>
+      hoverBackgroundColor?: Array<string>
```

Comment:
also supports `borderColor`, `borderWidth`, `hoverBorderColor` and `hoverBorderWidth`